### PR TITLE
Prepare to export LJM types

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: modules/connectivity/NetworkInfo.js [hunk 1]
```diff
@@ -26,7 +26,6 @@ export class NetworkInfo extends Listenable {
 
     /**
      * Updates the network info state.
-     * @param {boolean} isOnline - {@code true} if internet is online or {@code false} otherwise.
```
Comment (robertpin) [inline modules/connectivity/NetworkInfo.js:29]: Why was this removed?

Comment (saghul) [reply modules/connectivity/NetworkInfo.js:29]: It's incorrectly defined as a Boolean not an object containing a Boolean. TS guesses it right.

**Review by robertpin — APPROVED** [review]
